examples/inverse-calculations.py

```diff
@@ -0,0 +1,63 @@
+#%%
+import resdx
+import numpy as np
+from scipy import optimize
+from scipy.optimize import fsolve
+import random
+import matplotlib.pyplot as plt
+import seaborn as sns
+sns.set()
+
+def plot(x, y, xlabel, ylabel):
+    plt.figure()
+    plt.plot(x, y)
+    plt.xlabel(xlabel)
+    plt.ylabel(ylabel)
+    plt.savefig(f'output/{ylabel.replace(" ", "")}_vs_{xlabel.replace(" ", "")}.png')
```

Comment:
Let's make the file name a separate argument so it can be cleaner and just call them: "cooling-cop-v-seer.png" and "heating-cop-v-hspf.png".